smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -102,25 +131,91 @@ contract MixinKeys is
   view 
   {
     require(
-      _ownerOf[_tokenId] != address(0), 'NO_SUCH_KEY'
+      _keys[_tokenId].expirationTimestamp != 0, 'NO_SUCH_KEY'
     );
   }
 
+
   /**
-   * Get a key owned by a specific address
-   * @return The key owned by `_keyOwner`
+  * Migrate data from the previous single owner => key mapping to 
+  * the new data structure w multiple tokens.
+  * @dev when all record schemas are sucessfully upgraded, this function will update the `schemaVersion`
+  * variable to the latest/current lock version
   */
-  function getKeyByOwner(
-    address _keyOwner
+  function migrate() public {
```

Comment:
So we actually need to pass a value here as a blob so we can re-use it. If we don't pass a value this will revert for locks with too many keys.